dask_cuda/benchmarks/local_cudf_merge.py

```diff
@@ -232,6 +242,12 @@ def parse_args():
         type=str,
         help="Write dask profile report (E.g. dask-report.html)",
     )
+    parser.add_argument(
```

Comment:
Should `set-index` but the default setting ?

Reply:
Yeah - I wasn't sure about this.  To compare with the c++/ucx numbers, we probably do need the set_index call (or something like it).  But since dask doesnt actually need to sort the base dataframe to perform the merge, I also think the non-sorted results are pretty valuable.